conf/rwahsNavigation.conf

```diff
@@ -0,0 +1,33 @@
+enabled = 1
+
+new_menu_shortcuts = {
+	library = library,
```

Comment:
Looking at this list now after merging the code PR, the keys and values seem redundant - arguably it could be:
`new_menu_shortcuts = [artwork, costume, museumArtefact, photograph, publicMemorial]`
but I think that horse has already bolted (and tests were written for it that measured how far it had bolted).